src/rules.json

```diff
@@ -78,9 +78,9 @@
             "severity": "Error"
         },
         {
-            "name": "bad opernds for .word directive",
-            "pattern": "\\.word[\\s]+(\\S+)",
-            "validPattern": "^(-?\\d+|0x[a-fA-F0-9]+|'\\\\?.'|[a-zA-Z_$@][a-zA-Z0-9_$@]*[+-]?[-?\\d+]?)$",
+            "name": "bad operand for .word directive",
+            "pattern": "\\.word[\\s]+('.*'|\\\".*\\\"|'?\\w+'?)(?:(?:[ ;\\n])|$)",
+            "validPattern": "^(-?\\d+|0x[a-fA-F0-9]+|'\\\\.'|''|'.'|[a-zA-Z_$@][a-zA-Z0-9_$@]*[+-]?[-?\\d+]?)$",
```

Comment:
While this later was superceded by more specific rules, this was intended to capture single and double quote encapsulated strings, included those with slash escape characters inside.